### PR TITLE
Pods 7818

### PR DESCRIPTION
**Pull request by grantrobson**

### CODE REVIEW

File: app/controllers/mccloud/IsPublicServicePensionsRemedyController.scala [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+          case Some(value) => form(chargeTypeDescription).fill(value)
+        }
+
+        val json = Json.obj(
+          "srn" -> srn,
+          "startDate" -> Some(localDateToString(startDate)),
+          "form" -> preparedForm,
+          "viewModel" -> viewModel,
+          "radios" -> Radios.yesNo(preparedForm("value")),
+          "chargeTypeDescription" -> Messages(s"isPublicServicePensionsRemedy.chargeType.${chargeType.toString}")
```
Comment (ngmjohnstone) [inline app/controllers/mccloud/IsPublicServicePensionsRemedyController.scala:91]: Could you use the val you created on 72 here as well?

File: app/controllers/mccloud/IsPublicServicePensionsRemedyController.scala [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+              returnUrl = controllers.routes.ReturnToSchemeDetailsController.returnToSchemeDetails(srn, startDate, accessType, version).url,
+              schemeName = schemeName
+            )
+            val json = Json.obj(
+              "srn" -> srn,
+              "startDate" -> Some(localDateToString(startDate)),
+              "form" -> formWithErrors,
+              "viewModel" -> viewModel,
+              "radios" -> Radios.yesNo(formWithErrors("value")),
+              "chargeTypeDescription" -> Messages(s"isPublicServicePensionsRemedy.chargeType.${chargeType.toString}")
```
Comment (ngmjohnstone) [inline app/controllers/mccloud/IsPublicServicePensionsRemedyController.scala:123]: Same as previous

File: app/controllers/mccloud/IsPublicServicePensionsRemedyController.scala (hunk 1 above)
Comment (grantrobson) [reply app/controllers/mccloud/IsPublicServicePensionsRemedyController.scala]: Well spotted. I've fixed this.

File: app/controllers/mccloud/IsPublicServicePensionsRemedyController.scala (hunk 2 above)
Comment (grantrobson) [reply app/controllers/mccloud/IsPublicServicePensionsRemedyController.scala]: Fixed

**Pull request merged**